packages/react/src/experimental/Forms/EntitySelect/CreateItem/index.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from "@storybook/react-vite"
+
+import { ComponentProps } from "react"
+import { fn } from "storybook/test"
+import { CreateItem } from "./index"
+
+const meta: Meta = {
```

Comment:
disable snapshot test either for this one (if you make the `WithCreate` story open by default) or the `WithCreate` story

this way we don't create several snapshots of the same thing